### DISCUSSION TITLE
Product Specific Description field for products with Combinations

### DISCUSSION THREAD

**Discussion by num47** [PrestaShop/PrestaShop#40174] [Ideas]
#### Is your feature request related to a problem?
Yes

I have products with two-three hundred combinations. It becomes impossible to give separate descriptions or add few extra large images in the description box for all those different combinations. 

#### Describe the solution you'd like

If there were an option for a description field that can be added to a specific product combination that will only be loaded if that combination were selected by the customer then it would be great. 

#### Alternatives you've considered

Couldn’t find any.

#### Additional context

None.

**Comment by florine2623** [comment]
Hello @num47 ,

Thank you for your suggestion. The Product Team will take it into consideration for future developments about the Description field. 
However for the images you can choose a specific image for each combination.

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know!

Thank you

**Comment by florine2623** [comment]
ping @PrestaShop/product-team ,

What do you think about @num47 's idea of adding a "Description field" for each combination ?

Thanks!
